plutus-metatheory/changelog.d/20230420_135648_mjaskelioff_untypedbuiltins.rst

```diff
@@ -0,0 +1,34 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. Removed
+.. -------
+..
+.. - A bullet item for the Removed category.
+..
+.. Added
+.. -----
+..
+Support for all V2 builtin-functions for UPLC (mode U)
```

Comment:
I think (I hope) I fixed it...

Reply:
We're not doing changelogs for `plutus-metatheory` since it doesn't get released. Hence it is lacking the configuration that tells `scriv` to use markdown here.

Reply:
This PR also changes some golden files in the conformance test suite, which are very much user-facing. I wonder if that counts as something that is being released.

Reply:
We're not releasing it :shrug: at the moment we only ever assemble changelogs when we release a package...

Reply:
*something that is going to be released (in some official capacity)

Reply:
ok. Then I will not bother adding changelogs (not that I added many of them).